### PR TITLE
Update anchor-links.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/web/html/links/anchor-links.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/web/html/links/anchor-links.md?ref=c86c3a1aa2f80fcafae7be6ca3a682d6797af6c8)

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +48,11 @@ Types of change:
 
 ### Fixed
 
+## February 14th 2022
+
+### Changed
+-[Web - Anchor Links - Make RQ clearer](https://github.com/enkidevs/curriculum/pull/3041)
```
Comment (kapnobatai137) [inline CHANGELOG.md:54]:
```suggestion
- [Web - Anchor Links - Make RQ clearer](https://github.com/enkidevs/curriculum/pull/3041) 
```

File: web/html/links/anchor-links.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ Link to the
 
 ## Revision
 
-What denotes an anchor link?
+Add the missing code to create an anchor link that navigates to the portion of the page whose id is `contact`:
```
Comment (kapnobatai137) [inline web/html/links/anchor-links.md:82]: what do you think about this?

```suggestion
Create an anchor link that navigates to the HTML element with the `contact` id:
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Stefan-Stojanovic) [reply CHANGELOG.md]: 👍

File: web/html/links/anchor-links.md (hunk 2 above)
Comment (Stefan-Stojanovic) [reply web/html/links/anchor-links.md]: Sounds good

**Review by kapnobatai137 — DISMISSED** [review]

**Pull request merged**